frontend/components/ActivityDetails/InstallDetails/VppInstallDetailsModal/VppInstallDetailsModal.tsx

```diff
@@ -112,23 +117,67 @@ export const getStatusMessage = ({
 
   // Verification failed (timeout)
   if (displayStatus === "failed_install" && isMDMStatusAcknowledged) {
+    const isMacOS = platform === "darwin";
+    const isApplePlatform = ["darwin", "ios", "ipados"].includes(
+      platform || ""
+    );
```

Comment:
There is already some platform helper methods:
https://github.com/fleetdm/fleet/blob/aa3cae53e7ad52e12a00df57c85dc4ad72bdbcac/frontend/interfaces/platform.ts#L143-L153